packages/automation/src/mapping.ts

```diff
@@ -5,6 +5,51 @@ import {
   ParamDefinition,
 } from './types';
 
+export const commandTypeJsonMapping: Record<CommandContractType, string[]> = {
+  [CommandContractType.CloseCommand]: ['cdpId', 'triggerType', 'collRatio'],
+  [CommandContractType.BasicBuyCommand]: [
+    'cdpId',
+    'triggerType',
+    'execCollRatio',
+    'targetCollRatio',
+    'maxBuyPrice',
+    'continuous',
+    'deviation',
+    'maxBaseFeeInGwei',
+  ],
+  [CommandContractType.BasicSellCommand]: [
+    'cdpId',
+    'triggerType',
+    'execCollRatio',
+    'targetCollRatio',
+    'minSellPrice',
+    'continuous',
+    'deviation',
+    'maxBaseFeeInGwei',
+  ],
+  [CommandContractType.AutoTakeProfitCommand]: [
+    'cdpId',
+    'triggerType',
+    'executionPrice',
+    'maxBaseFeeInGwei',
+  ],
+  [CommandContractType.SimpleAAVESellCommand]: [
+    'positionAddress',
+    'triggerType',
+    'amount',
+    'interval',
+    'recipient',
+  ],
+  [CommandContractType.AaveStopLossCommand]: [
+    'positionAddress',
+    'triggerType',
+    'collateralToken',
+    'debtToken',
+    'ltv',
```

Comment:
```suggestion
    'slLevel',
```

Reply:
that's name for FE and it do not need to match SC, it was requested by Seba